tests/boardfarm_plugins/boardfarm_prplmesh/tests/initial_ap_config.py

```diff
@@ -0,0 +1,68 @@
+"""
+SPDX-License-Identifier: BSD-2-Clause-Patent
+SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+This code is subject to the terms of the BSD+Patent license.
+See LICENSE file for more details.
+"""
+
+import boardfarm
+
+from boardfarm.tests import bft_base_test
+from environment import sniffer
+
+
+class InitialApConfig(bft_base_test.BftBaseTest):
+    '''
+    Check initial configuration on device
+    '''
+
+    def runTest(self):
+        regex_list = [
+            r"WSC Global authentication success",
+            r"KWA \(Key Wrap Auth\) success",
+            r".* Controller configuration \(WSC M2 Encrypted Settings\)"
+        ]
+        failed_regex = []
+
+        wired_sniffer = sniffer.Sniffer(self.dev.board.gw_iface, True, boardfarm.config.output_dir)
+        wired_sniffer.start(self.__class__.__name__)
+
+        for radio in self.dev.board.al_entity.radios:
+            for regex in regex_list:
```

Comment:
Although this looks like a nice refactoring at first sight, I don't really agree. I think it's harder to read this way than when it's explicitly separate checks, like is done in test_flows.

In general, try to take over the code from test_flows as literally as possible, and do any refactorings in a second stage.